src/arrayMethodSort.js

```diff
@@ -4,8 +4,31 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => {
+      if (a.toString() < b.toString()) {
+        return -1;
+      }
+
+      if (a.toString() > b.toString()) {
+        return 1;
+      }
+
+      return 0;
+    },
+  ) {
```

Comment:
You can simplify like this
```suggestion
  [].__proto__.sort2 = function (compareFunction = (a, b) => String(a) > String(b) ? 1 : -1)) {
```